org-roam-db.el

```diff
@@ -606,19 +607,12 @@ INFO is the org-element parsed buffer."
       (puthash (car row) (cadr row) ht))
     ht))
 
-(defun org-roam-db--file-hash (&optional file-path)
-  "Compute the hash of FILE-PATH, a file or current buffer."
-  ;; If it is a GPG encrypted file, we always want to compute the hash
-  ;; for the GPG encrypted file (undecrypted)
-  (when (and (not file-path) (equal "gpg" (file-name-extension (buffer-file-name))))
-    (setq file-path (buffer-file-name)))
-  (if file-path
-      (with-temp-buffer
-        (set-buffer-multibyte nil)
-        (insert-file-contents-literally file-path)
-        (secure-hash 'sha1 (current-buffer)))
-    (org-with-wide-buffer
-     (secure-hash 'sha1 (current-buffer)))))
+(defun org-roam-db--file-hash (file-path)
+  "Compute the hash of FILE-PATH."
+  (with-temp-buffer
+    (set-buffer-multibyte nil)
+    (insert-file-contents-literally file-path)
+    (secure-hash 'sha1 (current-buffer))))
```

Comment:
hmm i then wonder whether we really need to compute the content hash, or just get the hash of the file directly then. I forget why we needed to use the file contents

Reply:
There doesn't seem to be a way to compute the hash from the file directly, as `secure-hash` only works on a string or a buffer; I'm not sure if shelling out would be worth it either.

Reply:
yeah I think this is fine too.